tool/templates/dart/method.dart.jinja

```diff
@@ -0,0 +1,43 @@
+{%- if !m.docs.is_empty() %}
+/// {{m.docs}}
+{%- endif %}
+{{ m.declaration -}} { {##}
+		{%- for slice_conversion in m.slice_conversions %}
+		{%- if loop.first %}
```

Comment:
suggestion: I know it would get ugly to indent the actual code this way, but can we indent the jinja bits?